composer.json

```diff
@@ -8,35 +8,26 @@
             "email": "info@palantir.net"
         }
     ],
-    "minimum-stability": "dev",
-    "prefer-stable": true,
-    "config": {
-        "platform": {
-            "php": "7.2"
-        },
-        "preferred-install": {
-            "*": "dist"
-        },
-        "sort-packages": true
-    },
     "repositories": [
         {
             "type": "composer",
             "url": "https://packages.drupal.org/8"
         }
     ],
     "require": {
-        "composer/installers": "^1.0",
-        "drupal/core-composer-scaffold": "^8.8",
+        "composer/installers": "^1.2",
+        "drupal/coder": "^8.3.6",
         "drupal/config_installer": "^1.8",
         "drupal/config_split": "^1.4",
-        "drupal/core-recommended": "^8.7.0",
+        "drupal/core-composer-scaffold": "^8.8",
+        "drupal/core-project-message": "^8.8",
```

Comment:
Can you link where I could find documentation on this package?

Reply:
See https://www.drupal.org/node/3090416 and https://packagist.org/packages/drupal/core-project-message

I'd prefer not to add this package, since the "plugin's only purpose is to print a message at the end of a composer create-project or a composer install". I don't think that's something we need here, especially since any message that we would add to the composer.json would be included in the composer.json of projects built off of this template.

Reply:
:100: